src/server/pegasus_server_impl.cpp

```diff
@@ -1297,7 +1303,10 @@ void pegasus_server_impl::on_get_scanner(get_scanner_rpc rpc)
         _pfc_recent_filter_count->add(filter_count);
     }
 
-    _cu_calculator->add_scan_cu(req, resp.error, resp.kvs);
+    // abandon calculate capacity unit
+    if (!request.only_return_count) {
```

Comment:
We should also calculate CU, and you have to update add_scan_cu, add some small CU at least.